JuiceMaker/JuiceMaker/Controller/ViewController.swift

```diff
@@ -7,12 +7,91 @@
+        } catch {
+            showNotEnoughStockAlert()
+        }
+    }
+    
+    private func showNotEnoughStockAlert() {
+        let alertBody = UIAlertController(title: MessageNameSpace.notEnoughStock, message: nil, preferredStyle: .alert)
+        let yesAction = UIAlertAction(title: MessageNameSpace.yes, style: .default, handler: {
+            _ in self.switchScreenToManageStockView()
+        })
+        let noAction = UIAlertAction(title: MessageNameSpace.no, style: .cancel, handler: nil)
+        alertBody.addAction(yesAction)
+        alertBody.addAction(noAction)
+        present(alertBody, animated: false, completion: nil)
+    }
+    
+    private func showJuiceReadyAlert(juiceName: String) {
+        let alertBody = UIAlertController(title: juiceName +  MessageNameSpace.juiceReady, message: nil, preferredStyle: .alert)
+        let confirmAction = UIAlertAction(title: MessageNameSpace.confirm, style: .default, handler: nil)
+        alertBody.addAction(confirmAction)
+        present(alertBody, animated: false, completion: nil)
+    }
+    
+    private func updateFruitStockLabel() {
+        stockOfStrawberryLabel.text = String(juiceMaker.fruitStore.fruits[FruitType.strawberry] ?? ConstantNameSpace.minimumNumberOfStock)
+        stockOfBananaLabel.text = String(juiceMaker.fruitStore.fruits[FruitType.banana] ?? ConstantNameSpace.minimumNumberOfStock)
+        stockOfPineappleLabel.text = String(juiceMaker.fruitStore.fruits[FruitType.pineapple] ?? ConstantNameSpace.minimumNumberOfStock)
+        stockOfKiwiLabel.text = String(juiceMaker.fruitStore.fruits[FruitType.kiwi] ?? ConstantNameSpace.minimumNumberOfStock)
+        stockOfMangoLabel.text = String(juiceMaker.fruitStore.fruits[FruitType.mango] ?? ConstantNameSpace.minimumNumberOfStock)
+    }
+    
+    private func updateFruitStockLabel(fruitType: [FruitType]) {
+        for fruit in fruitType {
+            switch fruit {
+            case FruitType.strawberry:
+                stockOfStrawberryLabel.text = String(juiceMaker.fruitStore.fruits[FruitType.strawberry] ?? ConstantNameSpace.minimumNumberOfStock)
+            case FruitType.banana:
+                stockOfBananaLabel.text = String(juiceMaker.fruitStore.fruits[FruitType.banana] ?? ConstantNameSpace.minimumNumberOfStock)
+            case FruitType.pineapple:
+                stockOfPineappleLabel.text = String(juiceMaker.fruitStore.fruits[FruitType.pineapple] ?? ConstantNameSpace.minimumNumberOfStock)
+            case FruitType.kiwi:
+                stockOfKiwiLabel.text = String(juiceMaker.fruitStore.fruits[FruitType.kiwi] ?? ConstantNameSpace.minimumNumberOfStock)
+            case FruitType.mango:
+                stockOfMangoLabel.text = String(juiceMaker.fruitStore.fruits[FruitType.mango] ?? ConstantNameSpace.minimumNumberOfStock)
+            }
+        }
+    }
+    
+    @IBAction func modifyFruitStockAction(_ sender: Any) {
+        switchScreenToManageStockView()
```

Comment:
[Naming] switch가 실제로 쓰이지는 않기 때문에 해당 네이밍은 조금 오해의 여지가 있을 수 있겠어요..!
VC를 새로 띄울때는 `show~`, `present~` 등의 단어들을 메서드에 많이 쓰는 것 같아요 ~!

```suggestion
        presentManageStockViewController()
```

Reply:
presentStockInventoryViewController로 변경했습니당!